corehq/apps/export/models/new.py

```diff
@@ -915,6 +924,77 @@ def get_filters(self):
         return []
 
 
+class SMSExportInstance(ExportInstance):
+    type = SMS_EXPORT
+    identifier = None
+
+    @classmethod
+    def _new_from_schema(cls, schema):
+        return cls(
+            domain=schema.domain,
+            tables=[
```

Comment:
another reason to define it there is so that when you add system properties, the export will automatically add those regardless of whether or not you have a saved version (handled in the `__insert_system_properties` function)